app/components/Configuration.js

```diff
@@ -125,11 +127,16 @@ export default class Configuration extends Component<Props> {
   }
 
   render() {
-    const { previousStage, configuration } = this.props;
-
+    const { previousStage, configuration, hideConfiguration } = this.props;
     const stage: StageDescription = stages[configuration.stage];
     const { stageValid, validationMessage } = this.checkStageValid();
 
+    if (this.isLaunching && configuration.policies.autostart) {
+      hideConfiguration();
+    }
+    // Only hide configuration (autostart) once
+    this.isLaunching = false;
```

Comment:
do i understand correctly that we're triggering a side effect *and* changing the instance during render here? if so, both are bad practice in react.
any reason not to move this to `componentDidMount`? i.e. instead of setting
```
this.isLaunching = true
```
just trigger 
```
if (configuration.policies.autostart) {
    hideConfiguration();
}
```

or am i missing some other lifecycle challenge here?